src/uu/nice/src/nice.rs

```diff
@@ -26,64 +26,49 @@ extern "C" {
     fn setpriority(which: c_int, who: c_int, prio: c_int) -> c_int;
 }
 
-pub fn uumain(args: impl uucore::Args) -> i32 {
-    let args = args.collect_str();
-
-    let mut opts = getopts::Options::new();
-
-    opts.optopt(
-        "n",
-        "adjustment",
-        "add N to the niceness (default is 10)",
-        "N",
-    );
-    opts.optflag("h", "help", "display this help and exit");
-    opts.optflag("V", "version", "output version information and exit");
-
-    let matches = match opts.parse(&args[1..]) {
-        Ok(m) => m,
-        Err(err) => {
-            show_error!("{}", err);
-            return 125;
-        }
-    };
-
-    if matches.opt_present("version") {
-        println!("{} {}", NAME, VERSION);
-        return 0;
-    }
-
-    if matches.opt_present("help") {
-        let msg = format!(
-            "{0} {1}
-
-Usage:
+fn get_usage() -> String {
+    format!(
+        "
   {0} [OPTIONS] [COMMAND [ARGS]]
 
 Run COMMAND with an adjusted niceness, which affects process scheduling.
 With no COMMAND, print the current niceness.  Niceness values range from at
 least -20 (most favorable to the process) to 19 (least favorable to the
 process).",
-            NAME, VERSION
-        );
+        executable!()
+    )
+}
 
-        print!("{}", opts.usage(&msg));
-        return 0;
-    }
+pub fn uumain(args: impl uucore::Args) -> i32 {
+    let usage = get_usage();
+
+    let matches = App::new(executable!())
+        .setting(AppSettings::TrailingVarArg)
+        .version(VERSION)
+        .usage(&usage[..])
+        .arg(
+            Arg::with_name("adjustment")
```

Comment:
For consistency with other binaries, please declare adjustment in a variable (or structure)